litgpt/pretrain.py

```diff
@@ -91,7 +91,9 @@ def setup(
     if model_config is not None and model_name is not None:
         raise ValueError("Only one of `model_name` or `model_config` can be set.")
     elif model_config is None and model_name is None:
-        model_name = "tiny-llama-1.1b"
+        from litgpt.config import name_to_config
```

Comment:
A local import isn't necessary here. Could you move it to the top of the file?

Reply:
Will do. It's been some time since I opened this PR and don't recall why I did this. I agree it's unnecessary and weird.